docs/index.html

```diff
@@ -100,12 +100,12 @@ <h1>markdown-it-docutils</h1>
 * H{sub}`2`O
 * 4{sup}`th` of July
 * {abbr}`CSS (Cascading Style Sheets)`
-          
```

Comment:
we can probably have prettier run against this?

Reply:
Yeah, changing these to yaml anyways!